src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -179,47 +418,54 @@ private void downloadService(V5VmOpenApiOrganizationService service) throws IOEx
   }
   
   private void downloadServiceChannels(File serviceFile) throws IOException {
-    V7VmOpenApiService service = new ObjectMapper().readValue(serviceFile, V7VmOpenApiService.class);
+    V8VmOpenApiService service = new ObjectMapper().readValue(serviceFile, V8VmOpenApiService.class);
     
-    List<V7VmOpenApiServiceServiceChannel> serviceChannels = service.getServiceChannels();
-    for (V7VmOpenApiServiceServiceChannel serviceChannel : serviceChannels) {
+    List<V8VmOpenApiServiceServiceChannel> serviceChannels = service.getServiceChannels();
+    for (V8VmOpenApiServiceServiceChannel serviceChannel : serviceChannels) {
       downloadServiceChannel(serviceChannel);
     }
   }
 
-  private void downloadServiceChannel(V7VmOpenApiServiceServiceChannel serviceChannel) {
+  private void downloadServiceChannel(V8VmOpenApiServiceServiceChannel serviceChannel) {
     String serviceChannelId = serviceChannel.getServiceChannel().getId().toString();
     File serviceChannelsFolder = getServiceChannelsFolder();
     
     String url = String.format("https://api.palvelutietovaranto.trn.suomi.fi/api/%s/ServiceChannel/%s", PTV, serviceChannelId);
     File target = new File(serviceChannelsFolder, String.format("%s.json", serviceChannelId));
     if (!target.exists()) {
       try {
-        download(url, target);          
-        System.out.println(url);
+        download(url, target, null);    
       } catch (IOException e) {
         e.printStackTrace();
       }
-    } else {
-      System.out.println(target.getAbsolutePath());
     }
   }
 
-  private void download(String url, File target) throws IOException, MalformedURLException, FileNotFoundException {
+  private void download(String url, File target, Class<?> outputClass) throws IOException, MalformedURLException, FileNotFoundException {      
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.net.MalformedURLException' which is a subclass of 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.FileNotFoundException' which is a subclass of 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)